### PR TITLE
TBRANDS-101 : Product Content, Part 2

### PR DESCRIPTION
**Pull request by ecarlisle**
## Description

Add "Size & Fit" and "Shipping & Returns" options as Product Content titles.

## Jira Ticket

- [TMEDIA-101](https://arcpublishing.atlassian.net/browse/TBRANDS-101)

## Acceptance Criteria

1. Product Content - Arc Block is available to be added to a page in Page Builder Editor.
2. The following types of Product Content are available for display from the Arc Commerce Content Source:
     a. Size & Fit - API Field - schema.sizeAndFit
     b. Shipping & Returns - API Field - schema.shippingAndReturns
3. A user within Page Builder can select the type of content to display. A user has the can select one of the following content types:
     a. Size & Fit
     b. Shipping & Returns
4. If a user selects the Size & Fit content type, the headline will display as Size & Fit
5. If a user selects the Shipping & Returns content type, the headline will display as Shipping & Returns
6. A user can override the headline by utilizing a custom field names Customize Headline.
     a. A tool tip exists and when a user clicks on it, it unveils the following message:
          i. By adding a custom headline, the default headline will be overridden.
7. As a user I can select if I want the Product Content - Arc Block to be expanded of collapsed on page load by utilizing a custom field/radio button that says: Collapsed on page load.
8. Product Content - Arc Block display and styling match designs.
9. Product Content - Arc Block is responsive and works across all device types.

## Test Steps

1. Checkout this branch for the blocks repo: `git checkout {TBRANDS-101-product-content}`
2. Run fusion feature pack with linked block `npx fusion start -f -l @wpmedia/product-content-block`
3. Create a new page in PageBuilder Editor using the "Right Rail" layout. Set the page Global Content with the following options:
     a. Content Source: `commerce-product`
     b. slug: `crocker-sandals`  
4. Add two instances of the "Product Content" Arc Block to the page in the `rightrail` section.
     a. Set the "Product Content" custom field of one to "Size & Fit"
     b. Set the "Product Content" custom field of the other to "Shipping & Returns"

## Effect Of Changes

You should see the following for the page using the above steps.
image
If the "Collapsed on Page Load" custom field is set to false for each block, the following will be seen:
image

### Before
The block's "Product Content" custom field did not have the option for "Size & Fit" or "Shipping & Returns".

### After
The block's "Product Content" custom field now has those two options. Content will be displayed from the respective schema fields for each option.

## Dependencies or Side Effects

Two phrases were added to the block's `intl.json` file in this pull request. Language-specific files in `/locale/` have been updated to reflect this.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [X] Confirmed all the test steps a reviewer will follow above are working.
- [X] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [X] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [X] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [X] Confirmed this PR has unit test files _(no additional tests were needed for changes in this ticket.)_.
  - [X] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [X] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/product-content-block/intl.json [hunk 1]
```diff
@@ -20,5 +20,27 @@
 		"no": "Details",
 		"pt": "Details",
 		"sv": "Details"
+	},
```
Comment (ecarlisle) [inline blocks/product-content-block/intl.json:23]: As mentioned in the PR Dependencies, phrase translations in this file were all English. I'd appreciate assistance in the process for getting the translations. I do have access to Lokalise. Thanks!

File: blocks/product-content-block/index.story.jsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const mockData = {
 };
 
 export const TitleAndContent = () => (
-	<ProductContentDisplay summary="Product Details">
+	<ProductContentDisplay summary={mockData.schema.productDetails.label}>
 		{mockData.schema.productDetails.value}
 	</ProductContentDisplay>
 );
```
Comment (ecarlisle) [inline blocks/product-content-block/index.story.jsx:30]: Do we want additional stories for cases like:
1. Using a custom title?
2. Displaying the details with a content initially expended?
Thanks!

File: blocks/product-content-block/index.story.jsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ const mockData = {
 };
 
 export const TitleAndContent = () => (
-	<ProductContentDisplay summary="Product Details">
+	<ProductContentDisplay summary={mockData.schema.productDetails.label}>
```
Comment (ecarlisle) [inline blocks/product-content-block/index.story.jsx:27]: Seemed better to get this value from the mock data provided.

Comment (matthewroach) [reply blocks/product-content-block/index.story.jsx:27]: As the component used in storybook is presentational, where the value comes from doesn't have a huge impact as the component will output the value of `summary`

**Review by matthewroach — APPROVED** [review]

**Pull request merged**